tests/Form/FormMapperTest.php

```diff
@@ -300,56 +300,56 @@ public function testIfFalseCombination(): void
 
     public function testIfTrueNested(): void
     {
-        $this->expectException(\LogicException::class);
-        $this->expectExceptionMessage('Cannot nest ifTrue or ifFalse call');
-
         $this->formMapper->ifTrue(true);
         $this->formMapper->ifTrue(true);
+        $this->formMapper->add('fooName');
```

Comment:
Shouldn't this (and the other tests) be more like:
```php
$this->formMapper
    ->ifTrue(true)
        ->ifTrue(true)
            ->add('fooName')
        ->ifEnd()
    ->ifEnd()
;
```

IMHO it's more understandable